config/config.go

```diff
@@ -16,7 +15,9 @@ import (
 
 // Config is the firebolt application configuration, including the configured source and all processing nodes
 type Config struct {
+	Version         string              `yaml:"version"`
```

Comment:
what is version for here?

Reply:
I will remove this, originally I had planned creating versions of config to handle both styles of yml configuration.